examples/classification/convert_from_original/convert_vgg.py

```diff
@@ -0,0 +1,37 @@
+import chainer
```

Comment:
Putting a model-specific utility under task-specific directory seems against your intention. The conversion code for VGG-16 can be used for other task than classification.

> This is because unlike other examples, VGG does not have its own demo.py or train.py

You won't add any training codes for VGG or other classification networks?

Anyway, we should keep consistency with other examples. If you want to reduce the number of subdirectories, how about putting detection models under `detection`?